android/src/main/java/com/hieuvp/fingerprint/ReactNativeFingerprintScannerModule.java

```diff
@@ -68,8 +68,14 @@ private String getErrorMessage() {
     }
 
      private boolean checkFingerPrintEnrolled() {
+         if(Build.VERSION.SDK_INT<23){
```

Comment:
Here, Our app supports from API  23 onwards to up, I think here we don't need to worry about version check